cpp-client/deephaven/client/include/public/deephaven/client/client_options.h

```diff
@@ -0,0 +1,192 @@
+/*
+ * Copyright (c) 2016-2022 Deephaven Data Labs and Patent Pending
+ */
+#pragma once
+
+#include <string>
+#include <utility>  // std::pair
+#include <vector>
+
+namespace deephaven::client {
+
+class Client;
+ 
+/**
+ * The ClientOptions object is intended to be passed to Client::connect(). For convenience, the mutating methods can be
+ * chained. For example:
+ * auto client = Client::connect("localhost:10000", ClientOptions().setBasicAuthentication("foo", "bar").setSessionType("groovy")
```

Comment:
I think I'm the author of this comment, but "while you're here" can you delete "For example:" and use @example as in

```
 * The ClientOptions object is intended to be passed to Client::connect(). For convenience, the mutating methods can be
 * chained.
 * @example auto client = Client::connect("localhost:10000", ClientOptions().setBasicAuthentication("foo", "bar").setSessionType("groovy")
```